### ISSUE TITLE
develop branch

### ISSUE THREAD

**Issue by sam4215** [FoundationDriven/FoundationAPI#1]
All other FoundationDriven projects have a master and a develop branch. Can we do the same here? I would but I'm on my phone.

**Comment by exstar** [comment]
Yes, Gitflow was already used here, but i accidentally committed to the master branche. From now, the develop branche will be used along with branches for features, bugfixes etc.
